.rustfmt.toml

```diff
@@ -1,4 +1,5 @@
-hard_tabs = true
+hard_tabs = false
```

Comment:
:eyes: :)

Reply:
It's mainly just that subxt, scale-decode and scale-encode all use spaces :)

But also, a really annoying thing I run into with tabs is making doc examples; I can't remember the exact issue but something like the IDE tries to make things tabs but cargo complains about tabs in doc comments; either way it all "just works" with spaces I guess (and spaces also seem to be the default anyway)